kalite/distributed/static/js/distributed/distributed-server.js

```diff
@@ -77,6 +80,75 @@ var StatusModel = Backbone.Model.extend({
         return (new Date(new Date() - this.get("client_server_time_diff"))).toISOString().slice(0, -1);
     },
 
+    login: function(username, password, facility, callback) {
+        // Add a callback to allow functions calling this method to do things depending on the result of
+        // the login - failure, success, and particular errors that can be noted on the UI (such as incorrect username)
+        var self = this;
+
+        data = {
+            username: username || "",
+            password: password || "",
+            facility: facility || ""
+        };
+
+        $.ajax({
+            url: USER_URL + "login/",
+            contentType: 'application/json',
+            dataType: 'json',
+            type: 'POST',
+            data: JSON.stringify(data),
+            success: function(data, status, response) {
```

Comment:
Can you move this success callback into its own function? I think that was the style in the other backbone files we have.